util.go

```diff
@@ -90,7 +90,19 @@ func parseConfig(
 
 	logger.Debug().Type("target", target).Msg("decoding configuration into the target object")
 	//nolint:wrapcheck // error is already wrapped by DecodeInto
```

Comment:
Let's move this into the `if `statement below (line 95) and not wrap the returned error.

Reply:
You meant this: a7c327d?

Reply:
Yeah, that's it 👍 Thanks!